From f6f4790eba13beb53ee0a1e169fd75e04ec9eefa Mon Sep 17 00:00:00 2001
From: Jocelyn Turcotte <jocelyn.turcotte@digia.com>
Date: Thu, 25 Jul 2013 17:25:47 +0200
Subject: [PATCH] Build files necessary for touch and gestures.

Also guard the use of MessagePumpAuraX11 in events_x.cc. We need to
build it to get symbols depending on base::NativeEvent.
---
 content/content_browser.gypi | 2 +-
 ui/ui.gyp                    | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/content/content_browser.gypi b/content/content_browser.gypi
index 1e3485c..c2e5f18 100644
--- a/content/content_browser.gypi
+++ b/content/content_browser.gypi
@@ -1479,7 +1479,7 @@
         ['exclude', '^browser/geolocation/wifi_data_provider_linux\\.cc$'],
       ],
     }],
-    ['use_aura!=1 and OS!="win"', {
+    ['use_aura!=1 and use_qt!=1 and OS!="win"', {
       'sources!': [
         'browser/renderer_host/input/touchscreen_tap_suppression_controller.cc',
         'browser/renderer_host/ui_events_helper.cc',
diff --git a/ui/ui.gyp b/ui/ui.gyp
index 09c011c..4716f6c 100644
--- a/ui/ui.gyp
+++ b/ui/ui.gyp
@@ -708,7 +708,7 @@
             ['exclude', 'base/dragdrop/drag_utils_aura.cc'],
           ],
         }],
-        ['use_aura==0 and toolkit_views==0', {
+        ['use_aura==0 and use_qt==0 and toolkit_views==0', {
           'sources/': [
             ['exclude', '^base/gestures/*'],
           ]
@@ -886,7 +886,7 @@
             'base/cursor/cursor_loader_null.h',
           ],
         }],
-        ['toolkit_views==0', {
+        ['use_qt==0 and toolkit_views==0', {
           'sources!': [
             'base/x/events_x.cc',
             'events/event.cc',
-- 
1.8.4.2

